tests/e2e.rs

```diff
@@ -464,9 +466,37 @@ fn client_server_integration(
         first_entity_0,
         Some("Entity 0"),
         Some("Entity 0 description"),
+        None,
+        None,
+        None,
+        "Successfully updated profile",
+    )?;
+
+    profile(
+        server_url,
+        &first_api_key0,
+        first_entity_0,
+        "csv",
+        "Display name,Description,Organization,Location,Links\nEntity 0,Entity 0 description,null,null,"
+    )?;
+
+    profile(
+        server_url,
+        &second_api_key0,
+        first_entity_0,
+        "csv",
+        "Display name,Description,Organization,Location,Links\nEntity 0,Entity 0 description,null,null,"
+    )?;
+
+    update_profile(
+        server_url,
+        &first_api_key0,
+        first_entity_0,
+        None,
+        Some("Entity 0 NEW description"),
         Some("Entity 0 organization"),
-        Some("Entity 0 location"),
-        vec!["http://example.com"],
```

Comment:
Only because links are special/different from the other data types, should we still test them?

Reply:
You're right, I'll also test links